src/utils/optimizely.js

```diff
@@ -0,0 +1,65 @@
+export const EVENTS = {
+  ENROLLMENT_CLICK: 'enterprise_learner_portal_enrollment_click',
+}
+
+export const EXPERIMENT_0 = {
+  ID: 21468970242,
+  VARIANT_IDS: {
+    CONTROL: 21446530045,
+    VARIANT_1: 21458980337,
+  },
+}
+
+export const getOptimizelyProps = () => ({
+  src: process.env.ENVIRONMENT === 'production'
```

Comment:
[curious] What is the benefit of using the non-edX CDN version of the script? Also curious why we wouldn't want to keep the import in the index.html file here? As is, we will need to load all the application JavaScript in the browser before we can call this function, whereas it would load immediately in index.html.

Reply:
We only store the prod Optimizely script in the CDN.  For stage or dev, it's only available through the Optimizely route.

Agreed it'd be better in `index.html`, but I actually couldn't my local environment to read the darn `OPTIMIZELY_PROJECT_ID` environment variable, and saw that we were doing a similar thing in prospectus: https://github.com/edx/prospectus/blob/87b2220755d83c69eb41f78110a9e9b1ada0f44c/src/utils/thirdParty.js

Reply:
@iloveagent57 prospectus is different architecturally than MFEs. Prospectus is built with Gatsby, a static-site generator, where the resulting HTML is computed at build time. Our MFEs don't have any HTML pre-rendered except for what's explicitly defined in index.html since the resulting HTML is all determined at run-time after loading all the application and vendor bundles. 

Other MFEs also include it specifically [in the index.html file](https://github.com/openedx/frontend-app-learning/blob/c0c51a3028eeb25718a37f6cf6ce962dc6c59e5a/public/index.html#L9-L11).